canvas_modules/common-canvas/src/common-properties/controls/toggle/toggle.jsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2017-2022 Elyra Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { Toggle } from "carbon-components-react";
+import ValidationMessage from "./../../components/validation-message";
+import * as ControlUtils from "./../../util/control-utils";
+import { STATES } from "./../../constants/constants.js";
+import classNames from "classnames";
+import Tooltip from "./../../../tooltip/tooltip.jsx";
+import { v4 as uuid4 } from "uuid";
+
+
+class ToggleControl extends React.Component {
+	constructor(props) {
+		super(props);
+		this.id = ControlUtils.getControlId(this.props.propertyId);
+	}
+
+
+	handleChange(value) {
+		this.props.controller.updatePropertyValue(this.props.propertyId, value);
+	}
+
+	render() {
+		const value = this.props.value ? this.props.value : "";
+		const toggleControl = (<Toggle
+			id={this.id}
+			size="sm"
+			disabled={this.props.state === STATES.DISABLED}
+			toggled={Boolean(this.props.value)}
+			labelA={this.props.toggleofflabel}
+			labelB={this.props.toggleonlabel}
+			onToggle={this.handleChange.bind(this)}
+		/>);
+
+
+		let display = toggleControl;
+		if (this.props.tableControl) {
```

Comment:
This shouldn't be needed.  I don't see a reason to add a tooltip in a table cell for a toggle.  This was used in textfield since there could be an overflow.

Reply:
@nmgokhale fixed the error issues 
<img width="548" alt="image" src="https://user-images.githubusercontent.com/89609176/182446228-f803969f-6d4b-4052-a960-6dba1fe9d4ec.png">